src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -3141,6 +3141,16 @@ LeaveGroupOnLogout.Enabled = 0
 
 Group.Raid.LevelRestriction = 10
 
+#
+#    Group.RandomRollMaximum
+#
+#     The maximum value for use with the client '/roll' command.
```

Comment:
I'm not the greatest with phrasing, possibility of refer /roll $Group.RandomRollMaximum and/or /roll $minValue $Group.RandomRollMaximum